src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -1,307 +1,323 @@
 <div class="challenge-card">
-  <div class="ev-card-panel ev-md-container card-bt-margin">
-    <p *ngIf="!isActive"><strong class="fw-light">Sorry, the challenge is not
-        active.</strong></p>
-    <div *ngIf="isActive" class="row row-lr-margin">
-      <div class="col-md-11 col-sm-11 col-xs-10 col-lr-pad">
-        <span class="fs-20 fw-regular">Submission Guidelines</span>
-        <div *ngIf="isChallengeHost">
-          <div class="fw-semibold content">
-            <p><strong class="fw-regular">Note: </strong>Your submissions won't
-              be visible on leaderboard for
-              <strong class="fw-regular blue-text"> Public Challenge Phases
-              </strong> since you are a challenge host
-              for this challenge.<br> This feature is only for your testing
-              purposes.</p>
+    <div class="ev-card-panel ev-md-container card-bt-margin">
```

Comment:
Please remove the unnecessary changes from this file.

Reply:
sir i have corrected the whole file indentation with 2 space per indent .

Reply:
so it is the changes of those

Reply:
i have indented the file the same way it was previously please take a look now